lab_6_pipeline/pipeline.py

```diff
@@ -86,21 +163,36 @@ def __init__(self, position: int, text: str, tokens: list[ConlluToken]):
         """
         Initializes ConlluSentence
         """
+        self._position = position
+        self._text = text
+        self._tokens = tokens
+
+    def _format_tokens(self, include_morphological_tags: bool) -> str:
```

Comment:
consider moving adding `f'# sent_id = {self._position}\n# text = {self._text}\n'` to the `get_conllu_text` method so that the responsibility is distributed more evenly

Reply:
moved!